py-polars/polars/internals/dataframe/frame.py

```diff
@@ -1937,21 +1937,24 @@ def to_numpy(self) -> np.ndarray[Any, Any]:
         else:
             return out
 
-    def to_pandas(
-        self, *args: Any, date_as_object: bool = False, **kwargs: Any
+    def to_pandas(  # noqa: D417
+        self, *args: Any, use_pyarrow_extension_array: bool = False, **kwargs: Any
     ) -> pd.DataFrame:
         """
         Cast to a pandas DataFrame.
 
         This requires that :mod:`pandas` and :mod:`pyarrow` are installed.
-        This operation clones data.
+        This operation clones data, unless `use_pyarrow_extension_array=True`.
 
         Parameters
         ----------
-        args
-            Arguments will be sent to :meth:`pyarrow.Table.to_pandas`.
-        date_as_object
-            Cast dates to objects. If ``False``, convert to ``datetime64[ns]`` dtype.
+        use_pyarrow_extension_array
```

Comment:
Can we make this default to `True`?

Reply:
and we need to require pandas 1.5.x or higher.

Reply:
maybe you can disable it for them

```
In [7]: import sys

In [8]: (sys.version_info.major,  sys.version_info.minor)
Out[8]: (3, 10)

In [9]: import pandas as pd
pd
In [10]: pd.__version__
Out[10]: '1.5.3'
```

Reply:
Given @kylebarron comment, let's be conservative here and set it to `False`.

Reply:
IMO it would still be _ideal_ to have it set to `True`, but if you're supporting pandas pre 1.5, then users would get different responses depending on their pandas version, which seems confusing.

Reply:
Th pyarrow backed extension array implementation is considered experimental in the pandas 1.5.0 release:
https://pandas.pydata.org/docs/whatsnew/v1.5.0.html#native-pyarrow-backed-extensionarray

> This feature is experimental, and the API can change in a future release without warning.

So defaulting to it, might be too soon (even not considering the rest of the ecosystem).